test/setup/main.tf

```diff
@@ -21,6 +21,12 @@ locals {
     "production",
   ]
 
+  teams = [
+    "frontend-team",
```

Comment:
FYI @yliaog - There is a mix of `-` and `_` which is causing the current test failure.

Reply:
Yes, fixed, thanks.